v19.2/performance.md

```diff
@@ -0,0 +1,101 @@
+
+                                   | CockroachDB       | Amazon Aurora   
+-----------------------------------|-------------------|--------------
+Max Throughput                     | 631851 tpmC       | 12582 tpmC     
+Max Warehouses with Max Efficiency | 50000 Warehouses  | 1000 Warehouses
+Max Number of Rows                 | 24.9B             | 0.499B         
+Max Unreplicated Data              | 4TB               | 0.08TB         
+Machine type                       | c5d.4xlarge       | r3.8xl         
+
+Unlike Amazon Aurora, CockroachDB achieves this performance in [`SERIALIZABLE` isolation](demo-serializable.html), the strongest isolation level in the SQL standard. Like many other databases, Aurora selectively degrades isolation levels for performance, leaving workloads susceptible to fraud and data loss.
+
+To learn more about our comparison with Amazon Aurora, see this [blog post](https://www.cockroachlabs.com/blog/cockroachdb-2dot1-performance/).
+
+<!-- To try this out for yourself on your laptop [visit this docs page](https://www.cockroachlabs.com/docs/v19.1/training/performance-benchmarking.html#main-content). Or, if you want to reproduce the full results of TPC-C [visit this page](https://www.cockroachlabs.com/guides/cockroachdb-performance/). -->
+
+### Linear Scaling
+
+CockroachDB has **no theoretical scaling limit** and, in practice, can achieve near-linear performance passed more than 256 nodes. Because the TPC-C results above reflect leaps in scale, to test linear scaling, Cockroach Labs ran a simple benchmark named KV 95 (95% point reads, 5% point writes, all uniformly distributed) on AWS `c5d.4xlarge` machines:
+
+<img src="{{ 'images/v19.2/linearscale.png' | relative_url }}" alt="CRDB Linear Scale" style="max-width:100%" />
+
+This chart shows that adding nodes increases throughput linearly while holding p50 and p99 latency constant.
+
+## Throughput
+
+As mentioned above, Cockroach Labs believes TPC-C provides the most realistic and objective measure for OLTP throughput. In the real world, applications generate transactional workloads that consist of a combination of reads and writes, possibly with concurrency and likely without all data being loaded into memory. If you see benchmark results quoted in QPS, take them with a grain of salt, because anything as simple as a “query” is unlikely to be representative of the workload you need to run in practice.
+
+With that in mind, however, you can use [Sysbench](https://github.com/akopytov/sysbench) for straight-forward throughput benchmarking. For example, on a 3-node cluster of AWS `c5d.9xlarge` machines across AWS’s `us-east-1` region (availability zones `a`, `b`, and `c`), CockroachDB can achieve 118,000 inserts per second on the `oltp_insert` workload and 336,000 reads per second on the `oltp_point_select` workload:
+
+<img src="{{ 'images/v19.2/sysbench-throughput.png' | relative_url }}" alt="Sysbench Throughput" style="max-width:100%" />
+
```

Comment:
Please call out the concurrency. If this the below latency numbers of from the same run as this throughput number (which I'm hopeful they are) you should make that clear.

Reply:
Add a sentence